1.10/deploying-services/gpu.md

```diff
@@ -0,0 +1,169 @@
+    |-------------------------------+----------------------+----------------------+
+    | GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+    | Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+    |===============================+======================+======================|
+    |   0  Tesla M60           Off  | 0000:04:00.0     Off |                    0 |
+    | N/A   34C    P0    39W / 150W |     34MiB /  7679MiB |      0%      Default |
+    +-------------------------------+----------------------+----------------------+
+    ```
+
+    You will also see an entry for **GPU** in the DC/OS GUI on the **Configuration** tab for your service.
+
+## Docker-Based Application Definition
+In this example, an app is deployed with GPUs that specifies a Docker container and [DC/OS Universal Container Runtime (UCR)](/docs/1.10/deploying-services/containerizers/).  You must use UCR to run a containerized application that uses GPUs. To use the Universal Container Runtime, set the container type to `MESOS`.
+
+1.  Create an app definition named `docker-gpu-test.json`.
+
+    ```json
+    {
+        "id": "docker-gpu-test",
+        "acceptedResourceRoles":["slave_public", "*"],
+        "cmd": "while [ true ] ; do nvidia-smi; sleep 5; done",
+        "cpus": 1,
+        "mem": 128,
+        "disk": 0,
+        "gpus": 1,
+        "instances": 1,
+        "container": {
+          "type": "MESOS",
+          "docker": {
+            "image": "nvidia/cuda"
+          }
+        }
+    }
+    ```
+    
+1.  Launch your app using the DC/OS CLI:
+
+    ```bash
+    dcos marathon app add docker-gpu-test.json
+    ```  
+
+    After your service has deployed, check the contents of `stdout` to verify that the service is producing the proper output from the `nvidia-smi` command. You should see something like the following, repeated once every 5 seconds. Access the log [via the DC/OS CLI](/docs/1.10/monitoring/logging/quickstart/) or from the **Health** page for your service on the DC/OS dashboard.
+
+    ```
+    +------------------------------------------------------+
+    | NVIDIA-SMI 352.79     Driver Version: 352.79         |
+    |-------------------------------+----------------------+----------------------+
+    | GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+    | Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+    |===============================+======================+======================|
+    |   0  Tesla M60           Off  | 0000:04:00.0     Off |                    0 |
+    | N/A   34C    P0    39W / 150W |     34MiB /  7679MiB |      0%      Default |
+    +-------------------------------+----------------------+----------------------+
+    ```
+    
+    You will also see an entry for **GPU** on the **Configuration** tab of the page for your service.
+
+# Limitations
```

Comment:
@klueska so you can now set this in the `gpus_are_scarce` parameter, instead of the C++ example?